docs/Manifest.toml

```diff
@@ -87,6 +87,12 @@ version = "2.3.0"
 uuid = "0dad84c5-d112-42e6-8d28-ef12dabb789f"
 version = "1.1.1"
 
+[[deps.ArnoldiMethod]]
```

Comment:
can we remove the docs update from this PR? It was updated in the PR Alexis just merged, #668

Reply:
yes! ill rebase